pkg/controllers/pod/pod_controller.go

```diff
@@ -127,7 +129,7 @@ func (r *PodReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.R
 				log.Error(err, "Failed to update restore annotation on Pod", "Namespace", pod.Namespace, "Pod", pod.Name)
 				return err != nil
 			}, func() error {
-				return common.UpdateReconfigureNicAnnotation(r.Client, ctx, pod, "true")
+				return setReconfigureNicStatus(r.Client, ctx, pod)
```

Comment:
I saw you only modified the pod_controller, so the subnetport still holds the info in the annotation, right? I'm curious why we don't also move the subnetport's `ReconfigureNic` to status, to keep the pod and subnetport implementation consistent..

Reply:
The change from annotation to status is a fix for spherelet's permission issue. If we also change this for subnetport then we need to discuss it with vm operator/wcpsvc team again. I prefer to keep the current implementation unchanged as in 9.2 we will eventually replace it with the vif restore solution.